addons/sale_project/views/project_task_views.xml

```diff
@@ -165,7 +165,7 @@
                 <field name="sale_line_id"
                     groups="!sales_team.group_sale_salesman"
                     string="Sales Order Item"
-                    options='{"no_open": True}'
+                    options='{"no_open": True, "no_create": True}'
                     context="{'create': False, 'edit': False, 'delete': False}"
                     readonly="1"
```

Comment:
We should also remove `readonly="1"`, no? 🧐

Reply:
opps I missed it as context came in b/w :sweat: